lib/minitest/reporters/spec_reporter.rb

```diff
@@ -28,10 +38,20 @@ def report
 
       def record(test)
         super
-        test_name = test.name.gsub(/^test_: /, 'test:')
-        print pad_test(test_name)
-        print_colored_status(test)
-        print(" (%.2fs)" % test.time) unless test.time.nil?
+        if options[:show_order] == :before
+           print_colored_status(test) if options[:show_status]
+           test_name = test.name.gsub(/^test_(\d+_)?/, '  ')
```

Comment:
As above comment.  One version retains your old format, the other uses my new format. 

I'm not clued up or have the time to study possible formats ATM, so I catered "my" version to solving the output I have with my Minitest::Spec based tests. All of which end up with a number in the beginning of them.